pkg/target/host.go

```diff
@@ -189,10 +192,14 @@ func (h *Host) Connect(updateHostHealthStatus chan *HostStatus, attemptCount int
 	return true
 }
 
+// TODO (grzkv): The name does not say that it tries to re-connect
+// TODO (grzkv): Do we need to pass channel every time? can it be part of host?
```

Comment:
The channel is injected by the cluster into the hosts as a single source to listen for the status of all hosts within channel. It can be injected into the hosts when creating the hosts, but would require some refactoring.